frontend/src/components/AddTrip/create-new-trip.js

```diff
@@ -0,0 +1,17 @@
+
+/**
+ * Cleans form data and creates new Trip document in firestore.
+ *
+ * TODO(Issue #43): Create this function and any associated helpers.
+ */
+function createTrip(name, description, destination,
+                    startDate, endDate, collaborators) {
```

Comment:
Would it make more sense to make the argument an object of structure:
```
{ name: "name", description: "description", ...}
```

Then, if you're missing one of the fields, you can still use the function.

Reply:
Yeah that might be cleaner. Although, even if the user enters nothing, an empty string will still be passed so there is no case where a field should be missing.